code/game/objects/items/devices/powersink.dm

```diff
@@ -122,46 +133,67 @@
 				"<span class='hear'>You hear a click.</span>")
 			set_mode(CLAMPED_OFF)
 
+/// Removes internal heat and shares it with the atmosphere.
+/obj/item/powersink/proc/release_heat()
+	var/turf/our_turf = get_turf(src)
+	var/temp_to_give = internal_heat / FRACTION_TO_RELEASE
+	internal_heat -= temp_to_give
+	var/datum/gas_mixture/environment = our_turf.return_air()
+	var/delta_temperature = temp_to_give / environment.heat_capacity()
+	if(delta_temperature)
+		environment.temperature += delta_temperature
```

Comment:
can this bring air above the heat cap?

also, I'm a bit disappointed that putting a power sink in a cold area won't help combat its overheating problem.

also also, what happens if the power sink is in space or another kind of vacuum?

Reply:
1: It doesnt heat enough to matter
2 & 3: People would just vaccum the room and bypass it entirely. Not worth the effort.